src/openrct2/ride/Vehicle.h

```diff
@@ -251,7 +251,7 @@ enum {
     VEHICLE_ENTRY_FLAG_VEHICLE_ANIMATION = 1 << 23,             // Set on animated vehicles like the Multi-dimension coaster trains, Miniature Railway locomotives and Helicycles.
     VEHICLE_ENTRY_FLAG_RIDER_ANIMATION = 1 << 24,               // Set when the animation updates rider sprite positions
     VEHICLE_ENTRY_FLAG_25 = 1 << 25,
-    VEHICLE_ENTRY_FLAG_26 = 1 << 26,
+    VEHICLE_ENTRY_FLAG_XY_LOADING_POSITIONS = 1 << 26,
```

Comment:
The meaning of this isn't immediately clear. It could use a comment.

Reply:
Yeah sorry i was in a rush to save my work when i wrote this. I'm still working out how best to do the peep loading positions. There are two different structures deepening on the state of this flag. Can't decide how best to expose that in the vehicle structure.